cli/internal/template/crud/create/create.go

```diff
@@ -4,27 +4,46 @@ package create
 
 import (
 	"github.com/aerogear/charmil/cli/internal/factory"
+	"github.com/aerogear/charmil/cli/internal/flagutil"
+
 	"github.com/spf13/cobra"
 )
 
 type createOptions struct {
+	// Stores value of the `name` argument
+	name string
+
+	// Stores value of the `output` flag
 	outputFormat string
-	autoUse      bool
+	// Stores value of the `use` flag
+	autoUse bool
+
+	// Stores the valid regex for `name` argument validation
+	validNameRegexp string
 
 	// You can add more fields here according to your requirements
 }
 
 // GetCreateCommand returns a new command for creating {{.Singular}} instances.
 func GetCreateCommand(f *factory.Factory) *cobra.Command {
-	opts := &createOptions{}
+	opts := &createOptions{
+		validNameRegexp: `^[a-z]([-a-z0-9]*[a-z0-9])?$`,
```

Comment:
@wtrocki In the UseMatches rule(Validator), shall we do regex validation for flags as well? Currently we do this for `Use` only.